clients/spark/build.sbt

```diff
@@ -1,6 +1,6 @@
-lazy val projectVersion = "0.14.2"
+lazy val projectVersion = "0.15.0"
 version := projectVersion
-lazy val hadoopVersion = "3.2.1"
+lazy val hadoopVersion = "3.3.4"
```

Comment:
According to [emr-7.0.0](https://docs.aws.amazon.com/emr/latest/ReleaseGuide/emr-700-release.html) the version of hadoop is `3.3.6`, why did you set lower here?

Reply:
I saw in another place that 3.3.4 is the right version for 7.0.0, I was probably wrong